protoc-gen-elm/message.go

```diff
@@ -2,10 +2,32 @@ package main
 
 import (
 	"fmt"
+	"strings"
 
 	"github.com/golang/protobuf/protoc-gen-go/descriptor"
 )
 
+func mapEntries(inField *descriptor.FieldDescriptorProto, inMessage *descriptor.DescriptorProto) (isMap bool, keyFieldDescriptor *descriptor.FieldDescriptorProto, valueFieldDescriptor *descriptor.FieldDescriptorProto) {
+	isRepeated :=
```

Comment:
Could you add a link here to the documentation that you mentioned in the PR discussion? I did not know about it and I found it super useful!

Reply:
Great idea.  Done